bundles/org.openhab.binding.enocean/.classpath

```diff
@@ -28,5 +28,22 @@
 			<attribute name="maven.pomderived" value="true"/>
 		</attributes>
 	</classpathentry>
+	<classpathentry kind="src" path="target/generated-sources/annotations">
```

Comment:
What are those new entries about? Do we need them?

Reply:
I am not sure. VS Code added these entries. I will check if it they are really needed.

Reply:
I reverted the changes